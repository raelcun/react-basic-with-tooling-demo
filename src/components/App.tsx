import * as React from 'react';

type Props = { title: string; subtitle: string };

export const App: React.FC<Props> = ({ title, subtitle }) => (
    <div>
        <h1>{title}</h1>
        <h3>{subtitle}</h3>
    </div>
);
