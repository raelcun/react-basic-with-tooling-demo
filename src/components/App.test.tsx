import * as React from 'react';
import { App } from './App';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('App', () => {
    test('should match snapshot', () => {
        const component = shallow(<App title="myTitle" subtitle="mySubtitle" />);
        expect(toJson(component)).toMatchSnapshot();
    });
});
