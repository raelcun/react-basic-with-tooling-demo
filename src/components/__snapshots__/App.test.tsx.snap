// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match snapshot 1`] = `
<div>
  <h1>
    myTitle
  </h1>
  <h3>
    mySubtitle
  </h3>
</div>
`;
